### PR TITLE
Forms: Adding regex error

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This PR adds "INVALID EMAIL" error when email does not pass RegEx test.

In addition:
- minor confirmation popup height bug (when screen is half screen)
- code styling/import fix
- minor privacy policy margin increase
- adding NAME to fieldNames

### CODE REVIEW

File: components/home-page/newsletter-sign-up/NewsletterSignUp.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ import {
 } from './NewsletterSignUp.styles';
 import ArchModal from '../../arch-modal/ArchModal';
 import ConfirmationPopup from '../../confirmation-popup/ConfirmationPopup';
+import { fieldNames } from '../../../lib/utils';
```
Comment (jean-zhang) [inline components/home-page/newsletter-sign-up/NewsletterSignUp.js:17]: nice!

File: lib/hooks.js [hunk 2]
```diff
@@ -20,3 +20,7 @@ export const useMatchMedia = (query) => {
 
   return match;
 };
+
+export const useValidEmail = new RegExp(
```
Comment (jean-zhang) [inline lib/hooks.js]: Feels slightly out of place in this file?

**Review by jean-zhang — APPROVED** [review]
too good 👉👈

**Review by jzhzhang66 — APPROVED** [review]
nice!!

**Pull request merged**